controls/V-219164.rb

```diff
@@ -1,4 +1,6 @@
 # encoding: UTF-8
+min_fail_delay = input('min_fail_delay', value: 4000000,
```

Comment:
There is no need to override the shared input from the inxpec.yml.

you can just call the input in the control `input('min_fail_delay')` and then set its default value in the `inspec.yml` in the `inputs:` section.

This allows us to centerlize the inputs and if a change is needed we can update it in one place

This also allows us to easily do a global find and replace in the controls should we want to change an inputs name.

Reply:
if you have done this in other areas of the code and I missed it, please update it.

The only time I would use a local variable inside of a control is when I need to construct something from inputs or ask a question to the system.

Reply:
Fixed these two.  I think there's a couple more to address--came from SIMP examples. I'll fix those under separate PR

Reply:
A couple commits to do it right.  Not sure how they're queued by actions, but ignore the failures until the last push updates the PR.